.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Report a case where the product is not working as expected
+title: ''
+labels: type:bug
+assignees: 
+
+---
+
+**Environment (Required on creation):**
+ 
+**Description (Required on creation; please attach any relevant screenshots, stacktraces, log files, etc. to the ticket)**
+ 
+**Steps to reproduce (Required on creation)**
+    
+**Observed Behavior (Required on creation)** 
+    
+**Expected behavior (Required on creation)**
+    
+**Root Cause (Required on prioritization)**
+    
+**Solution Ideas (Optional)**
+    
+**Hints (Optional)**
+
+**Links:** 
```

Comment:
I agree that it should be consistent. I removed all the `(optional)` suffixes now because I think this is clear from the context anyway and less is more. From my observations most people aren't following these hints, but I think it still makes sense to have them as reminders for the important sections.